spec/attribute.dd

```diff
@@ -1015,7 +1047,10 @@ alias TP = __traits(getAttributes, foo);
 
 pragma(msg, TP); // prints tuple(3, 4, 7, (SSS))
 pragma(msg, TP[2]); // prints 7
+
+void main() {}
```

Comment:
It's more to stop the example being wrapped in `void main` automagically. It would be good if that didn't happen when SPEC_RUNNABLE_EXAMPLE_COMPILE is used (unlike _RUN).

Reply:
Specifically it is needed ATM here because SSS is declared with a UDA which won't parse inside `main`.

Reply:
Workaround no longer needed with #3561.